hudi-common/src/main/java/org/apache/hudi/exception/HoodieMetaSyncException.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.hudi.exception;
+
+public class HoodieMetaSyncException extends HoodieException {
```

Comment:
Can we include the list of syncs involved so we can easily handle different sync failures with custom logic?

Reply:
So the message will contain all the other syncs. They could be parsed out but I guess I could add them in a list at the beginning of the exception to make parsing easier.  For custom logic, I don't think it would be handled using the exception.